src/middleware/powered-by/index.test.ts

```diff
@@ -11,6 +11,9 @@ describe('Powered by Middleware', () => {
   app.use('/poweredBy2/*', poweredBy())
   app.get('/poweredBy2', (c) => c.text('root'))
 
+  app.use('/poweredByPatrickJS/*', poweredBy({ serverName: 'PatrickJS' }))
```

Comment:
Using your name `PatrickJS` is not proper for this test. It's good to use more general words. `foo` is okay though it's boring.